luigi/lock.py

```diff
@@ -42,7 +42,7 @@ def getpcmd(pid):
                 _, val = lines
                 return val
     else:
-        cmd = 'ps -xo pid,args'
+        cmd = 'ps x -wwo pid,args'
```

Comment:
Can you add a couple of lines of comments above this line as you find necessary? Example coming to my mind:

 * What OS versions this have been tested for
 * Links to various online man-pages for various OSes
 * Summarize what behaviour these set of parameters give and how it fits with luigi.

I mean, this is fully optional, I only want you to include what you think will make it easier for the next person dealing with this code line.